airbyte-integrations/connector-templates/source-python-http-api/source_{{snakeCase name}}/source.py.hbs

```diff
@@ -102,20 +148,42 @@ class Incremental{{properCase name}}Stream({{properCase name}}Stream, ABC):
         """
         return {}
 
-class SingleShotPotatoGun(Incremental{{properCase name}}Stream):
+
+class Employees(Incremental{{properCase name}}Stream):
     """
     TODO: Change class name to match the table/data source this stream corresponds to.
     """
 
     # TODO: Fill in the cursor_field. Required.
-    cursor_field = 'fill_in_this_stream_cursor_field'
+    cursor_field = 'start_date'
 
     def path(self, **kwargs) -> str:
         """
-        TODO: Override this method to define the path this stream corresponds to. E.g. if the url is https://potatogun.com/v1/single then this should
+        TODO: Override this method to define the path this stream corresponds to. E.g. if the url is https://example-api.com/v1/employees then this should
         return "single". Required.
         """
-        return "customers"
+        return "employees"
+
+    def stream_slices(self, stream_state: Mapping[str, Any] = None, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
```

Comment:
@sherifnada main change for stream slices. I decided putting it just in the incremental stream made the most sense. I can see arguments for placing this in the base incremental class as well. Both work for me so feel free to merge in once you decide what makes the most sense.

Reply:
Looks great!